ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -235,6 +244,13 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
         + cluster.clusterName + '/notebooks/' + nbName);
   }
 
+  private jupyterLabUrl(cluster: Cluster, nbName: string): string {
```

Comment:
Please manually test a few cases if you haven't already:

- can open/execute the demo notebooks
- can save/open notebook with space in the name
- can save/open notebook with `1%2B1`

Also could you add a screenshot of what this looks like in the iframe?

Reply:
Let me add some tests, and I will add the screenshot asap.

Reply:
I have manually tested that it can open and execute the demo notebooks (which also covers space in name). I will test the notebook with `1%2B1`